src/features/unified-share-modal/__tests__/UnifiedShareForm.test.js

```diff
@@ -225,6 +231,12 @@ describe('features/unified-share-modal/UnifiedShareForm', () => {
             expect(wrapper.exists('InlineNotice')).toBe(true);
         });
 
+        test('should render the upsell inline notice component when component is passed', () => {
+            const wrapper = getWrapper({ upsellInlineNotice: mockUpsellInlineNotice });
+            const upsellInlineNotice = wrapper.find('[data-testid="upsellinlinenotice"]');
```

Comment:
since you are wrapping the component that is being passed in, you can select by the `className` on the wrapping `div` instead of the `data-testid`